docs/guides/errors.md

```diff
@@ -0,0 +1,156 @@
+# Errors
+
+All the errors that Prism returns conform to [RFC7807](https://tools.ietf.org/html/rfc7807) - Problem Details for HTTP APIs, which means you'll always get a **JSON object** with the following properties:
+
+* `type` (string)
+* `title` (string)
+* `status` (number)
+* `detail` (string)
+
+This document enumerates all the possible errors returned by Prism and provides some further information on how to solve them.
+
+## Routing errors
+
+This class of errors is returned when Prism is trying to identify the right resource to use to respond to the provided HTTP Request.
+
+### NO_BASE_URL_ERROR
+**Message: Attempted to make a request to a server but neither baseUrl param provided nor servers defined in the spec**
+#### Returned Status Code: `400`
+**Explanation:** This error occurs when Prism is being used as a proxy and the current document is missing at least a server in the dedicated array (both globally and locally). The proxy functionality is currently disabled in Prism, so this error should never happen.
+
+---
+
+### NO_RESOURCE_PROVIDED_ERROR
+**Message: Route not resolved, no resource provided**
+#### Returned Status Code: `404`
+**Explanation:** This error occurs when the current document does not have any endpoint.
+
+##### Example
+
+```yaml
+openapi: 3.0.1
+paths:
+```
+
+`curl -X POST http://localhost:4010/pets`
+
+---
+
+### NO_PATH_MATCHED_ERROR
+**Message: Route not resolved, no path matched**
+#### Returned Status Code: `404`
```

Comment:
This error could do with an example as server validation is a little not confusing.